src/hotspot/cpu/ppc/assembler_ppc.hpp

```diff
@@ -2522,6 +2522,17 @@ class Assembler : public AbstractAssembler {
   // Load pointer d from s1+roc.
   void ld_ptr(Register d, RegisterOrConstant roc, Register s1 = noreg) { ld(d, roc, s1); }
 
+  void ld( Register d, Address &a);
+  void lwa( Register d, Address &a);
+  void lwz( Register d, Address &a);
+  void lha( Register d, Address &a);
+  void lhz( Register d, Address &a);
+  void lbz( Register d, Address &a);
+  void std( Register d, Address &a, Register tmp = noreg);
+  void stw( Register d, Address &a, Register tmp = noreg);
+  void sth( Register d, Address &a, Register tmp = noreg);
+  void stb( Register d, Address &a, Register tmp = noreg);
```

Comment:
Spaces before "Register d" are uncommon. Please remove them (you can keep the one for ld to align it with the other ones).

Reply:
Thank you Martin. I have fixed the alignment for both the cases (RegisterOrConstant and Address obj). Also I have adapted the same changes to load instructions.